packages/overlays/src/GlobalOverlayController.js

```diff
@@ -78,9 +82,24 @@ export class GlobalOverlayController {
    * Hides the overlay.
    */
   hide() {
+    if (this._expandedInvokerNode) {
+      this._expandedInvokerNode.setAttribute('aria-expanded', 'false');
+    }
     this._createOrUpdateOverlay(false, this._data);
   }
 
+  addInvokerNodes(nodes) {
+    nodes.forEach(node => {
+      node.addEventListener('click', () => {
+        node.setAttribute('aria-expanded', 'true');
+        this._expandedInvokerNode = node;
+        this.show(node);
+      });
+      node.setAttribute('aria-haspopup', 'dialog');
```

Comment:
Not sure if this applies to all global overlays. 
It could in theory also be 'listbox', 'grid', 'menu' or 'tree' (https://www.w3.org/TR/wai-aria/#aria-haspopup).
Maybe 'true' would be the safest option. @erikkroes, what do you think?